### ISSUE TITLE
Aqara Contact Sensors Only Report Open State

### ISSUE THREAD

**Issue by bigeazy000** [mdaskalov/homebridge-tasmota-zbbridge#16]
Hello,

I have started using this plugin with some zigbee outlets and Aqara door sensors. The outlets are working perfectly, however the Aqara contact sensors never report in a closed state. So, they appear to be constantly open. Here is an example of the console output on the Sonoff Taamota bridge:

01:26:34.695 MQT: tele/tasmota_E1890E/5FA7/SENSOR = {"ZbReceived":{"0x5FA7":{"Device":"0x5FA7","Name":"Garage Entry Door Sensor","Contact":1,"Endpoint":1,"LinkQuality":66}}}

01:26:38.485 MQT: tele/tasmota_E1890E/5FA7/SENSOR = {"ZbReceived":{"0x5FA7":{"Device":"0x5FA7","Name":"Garage Entry Door Sensor","Contact":0,"Endpoint":1,"LinkQuality":66}}}

What can I change to make the plugin process the second message? Thanks in advance for any help you can provide..

**Comment by mdaskalov** [comment]
The [ContactSensor](https://developers.homebridge.io/#/service/ContactSensor) can be used as Garage Door with following states (see [here](https://developers.homebridge.io/#/characteristic/ContactSensorState)):
```
CONTACT_DETECTED    0
CONTACT_NOT_DETECTED  1
```
if we use the Contact value your sensor should be able to detect `CONTACT_DETECTED ` then `CONTACT_NOT_DETECTED ` states.

The configuration should look like follows:
```
{
    "addr": "0x5FA7",
    "type": "sensor_0x0500_0xFFF_ContactSensor_ContactSensorState_Contact",
    "name": "Garage Entry Door Sensor"
}
```

I assume that the current state should be read using cluster 0x500 / attribute 0xFFF but I am not sure. 

This actually doesn’t matter, as the current implementation does not use these configuration settings to query the initial state of the sensor. The sensor is initially (after homebridge reboot) in "not reachable" state until the first event is received.

I will change this when I get some free time to make another release ;)

I’ve noticed that the value `0` was not  correctly evaluated in the sensor implementation and just published a new release with a fix. 

With the new release the contact sensor works as expected.

**Comment by bigeazy000** [comment]
Awesome, thank you! I will give the new version a shot and report back :)

I also noticed in the earlier testing that the sensors were in unreachable state until an event was triggered. I'm happy to test that too on my end once you put that fix in.

Thanks again..

**Comment by mdaskalov** [comment]
Resolved in #19
